homework_5/task_3.py

```diff
@@ -0,0 +1,14 @@
+with open('text.txt', 'r') as file:
+    the_text = file.read()
+
+test_str = the_text
+all_freq = {}
+
+for i in test_str:
+    if i in all_freq:
+        all_freq[i] += 1
+    else:
+        all_freq[i] = 1
+print(all_freq)
```

Comment:
your code counts all characters. The task is about letters only.
Also A and a is same letter